local.c

```diff
@@ -1198,7 +1197,7 @@ static char * get_short_attr_name(struct iio_channel *chn, const char *attr)
 		ptr += len + 1;
 
 	if (chn->name) {
-		size_t len = strlen(chn->name);
+		len = strlen(chn->name);
```

Comment:
the commit also needs a Signed-off-by tag;

Reply:
and preferably the same description as in the PR;
Github PRs don't track too well in git history; so the information in the PR description is sightly more useful in the git history to be able to view it with `git log` on the command line

Reply:
Ah yes, understood.